rest/route/subscription_routes.go

```diff
@@ -94,6 +102,22 @@ func (s *subscriptionPostHandler) ParseAndValidate(ctx context.Context, r *http.
 	return nil
 }
 
+func isSubscriptionAllowed(sub event.Subscription) (bool, string) {
+	for _, selector := range sub.Selectors {
+
+		if selector.Type == "object" {
```

Comment:
I think you can delete the first two if statements here, assuming that this function is only used for user created subscriptions

Reply:
The function is run for all subscriptions added through the route